bladex/blade.py

```diff
@@ -9,68 +9,46 @@
 class Blade(object):
     """
     Bottom-up parametrized blade construction.
-
```

Comment:
Are you sure that (some of) the blank lines are not required for properly generate documentation?
In any case, it is not very useful changing 30 lines of code (actually docstrings) just for avoiding blank lines

Reply:
Thank you for pointing it out. I'm going to reintroduce the original blank lines.